mcs/class/System.Transactions/Test/TransactionScopeTest.cs

```diff
@@ -1114,6 +1114,18 @@ public void ExplicitIsolationLevel()
 				Assert.AreEqual(IsolationLevel.ReadCommitted, Transaction.Current.IsolationLevel);
 			}
 		}
+
+		[Test]
+		[ExpectedException(typeof(TransactionAbortedException))]
+		public void TransactionScopeTimeout()
+		{
+			Assert.IsNull(Transaction.Current, "Ambient transaction exists (before)");
+			using (var ts = new TransactionScope(TransactionScopeOption.Required, TimeSpan.FromMilliseconds(1)))
+			{
+				for(int i = 0; i < 1000000000; i += 2) i--;
```

Comment:
I don't understand what this loop does?

Reply:
I was trying to do a loop that would take longer than the specified timeout.

The obvious way would have been referencing System.Threading and sleeping the thread instead of looping. I tried looking at the tests structure but couldn't understand what I should do to reference the Thread class, any help there is greatly appreciated :)

Reply:
@poloBBQ adding `using System.Threading;` to the top of the file should be enough. Let me know if you run into issues :)

Reply:
You're right, I was trying to use the namespace explicitly (i.e. System.Threading.Thread.Sleep(...)) and was getting an error when running the tests
`error CS0234: The type or namespace name 'Threading' does not exist in the namespace 'MonoTests.System' (are you missing an assembly reference?)`

Much better like this!

I think all your suggestions are accepted and this change committed properly, not sure everything is right as this is the first PR review I receive.

Reply:
Looks good, thanks